sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs

```diff
@@ -303,11 +303,9 @@ private TestRunShardDto GetTestRunEndShard(TestRunCompleteEventArgs e)
 
             var result = FailedTestCount > 0 ? TestCaseResultStatus.s_fAILED : TestCaseResultStatus.s_pASSED;
 
-            if (e.ElapsedTimeInRunningTests != null)
```

Comment:
The local variable 'e' is of type TestRunCompleteEventArgs which is from Microsoft.VisualStudio.TestPlatform.ObjectModel.Client which is part of the Visual Studio Test Platform. That type ElapsedTimeInRunningTests is not nullable in .NET 8.  To be safe I'll put this back and suppress the warning, since in netstandard2.0 it could be null.